cmd/server/main.go

```diff
@@ -5,10 +5,12 @@ import (
 
 	diskapi "github.com/kubernetes-csi/csi-proxy/internal/os/disk"
 	filesystemapi "github.com/kubernetes-csi/csi-proxy/internal/os/filesystem"
+        smbapi "github.com/kubernetes-csi/csi-proxy/internal/os/smb"
```

Comment:
Please fix up the indentation here.

Reply:
fixed.

Reply:
Sorry - just one remaining nit: can you please fix up the above line [8] for `smbapi`? Thanks for fixing `smbsrv` below!

Reply:
sorry I missed it. fixed